src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -141,81 +142,55 @@ private static void GetCurrentEditorConfigOptionsCSharp(OptionSet optionSet, Str
             CSharpFormattingOptions_GenerateEditorconfig(optionSet, CSharpFormattingOptions.SpaceAfterMethodCallName, editorconfig);
             // csharp_space_between_method_call_empty_parameter_list_parentheses
             CSharpFormattingOptions_GenerateEditorconfig(optionSet, CSharpFormattingOptions.SpaceBetweenEmptyMethodCallParentheses, editorconfig);
-
             editorconfig.AppendLine();
+
             editorconfig.AppendLine("# Wrapping preferences:");
             // csharp_preserve_single_line_statements
             CSharpFormattingOptions_GenerateEditorconfig(optionSet, CSharpFormattingOptions.WrappingKeepStatementsOnSingleLine, editorconfig);
             // csharp_preserve_single_line_blocks
             CSharpFormattingOptions_GenerateEditorconfig(optionSet, CSharpFormattingOptions.WrappingPreserveSingleLine, editorconfig);
         }
 
-        private static void CSharpFormattingOptions_GenerateEditorconfig(OptionSet optionSet, Option<bool> option, StringBuilder editorconfig)
-        {
-            var element = option.StorageLocations.OfType<EditorConfigStorageLocation<bool>>().FirstOrDefault();
-            if (element != null)
-            {
-                editorconfig.Append(element.KeyName + " = ");
-
-                editorconfig.AppendLine(optionSet.GetOption(option).ToString().ToLowerInvariant());
-            }
-        }
-
         private static void CSharpCodeStyleOptions_GenerateEditorconfig(OptionSet optionSet, PerLanguageOption<CodeStyleOption<bool>> option, StringBuilder editorconfig)
         {
             var element = option.StorageLocations.OfType<EditorConfigStorageLocation<CodeStyleOption<bool>>>().FirstOrDefault();
             if (element != null)
             {
-                editorconfig.Append(element.KeyName + " = ");
+                GridOptionPreviewControl.AppendName(element.KeyName, editorconfig);
```

Comment:
it looks really weird to have this be `GridOptionPreviewControl.AppendName`.  Is this because you have helpers somewhere you're sharing?  If so, consider putting in a static class and hten just do a static import of that class.